index.bs

```diff
@@ -318,7 +319,7 @@ Get response end time algorithm {#sec-get-resonse-end}
 
 <div algorithm="PerformanceElementTiming responseEnd">
 When asked to <dfn>get response end time</dfn> with {{Request}} |request| as input, run these steps:
-    1. If the |request| is <code>null</code>, return 0.
+    1. If the |request| is <code>null</code> or associated to a data URL, return 0.
```

Comment:
Instead of "associated to a data URL", maybe "its scheme is 'data'" or somesuch?